### PR TITLE
Feat/accept all invites

### PR DESCRIPTION
**Pull request by sviatosluv**

### CODE REVIEW

File: lib/chat_web/live/main_live/page/dialog.ex [hunk 1]
```diff
@@ -280,26 +280,34 @@ defmodule ChatWeb.MainLive.Page.Dialog do
...
+        content: content
+      }) do
+    new_room_identity =
+      content
       |> StorageId.from_json()
       |> RoomInvites.get()
       |> Identity.from_strings()
 
-    if rooms |> Enum.any?(&(&1.private_key == new_room_identitiy.private_key)) do
+    if rooms |> Enum.any?(&(&1.private_key == new_room_identity.private_key)) do
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/page/dialog.ex:293]: please rewrite using `room_map`

**Comment by sviatosluv** [comment]
@sergey-lukianov should an invite update be broadcasted to the channel?

File: lib/chat_web/live/main_live/page/dialog.ex [hunk 2]
```diff
@@ -280,28 +281,37 @@ defmodule ChatWeb.MainLive.Page.Dialog do
...
+        render_fun
+      ) do
+    new_room_identity =
+      content
       |> StorageId.from_json()
       |> RoomInvites.get()
       |> Identity.from_strings()
 
-    if rooms |> Enum.any?(&(&1.private_key == new_room_identitiy.private_key)) do
+    if Identity.pub_key(new_room_identity) in Map.keys(room_map) do
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/page/dialog.ex:295]: Map.has_key?

File: lib/chat_web/live/main_live/page/dialog.ex [hunk 3]
```diff
@@ -311,7 +321,7 @@ defmodule ChatWeb.MainLive.Page.Dialog do
       |> RoomInvites.get()
       |> Identity.from_strings()
 
-    if rooms[new_room_identity.public_key].private_key == new_room_identity.private_key do
+    if Identity.pub_key(new_room_identity) in Map.keys(room_map) do
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/page/dialog.ex:324]: Map.has_key?

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**